Clarify ESENT settings for SC audit instances (#6220)
Particular/docs.particular.net: servicecontrol/audit-instances/creating-config-file.md

#### Raven/Esent/LogsPath

> The path for the Esent logs on disk. Note that this setting applies only to audit instances upgraded from previous versions of ServiceControl and that still use RavenDB 3.5. New audit instances use RavenDB 5 and don't rely on ESENT.

Reviewer: Align it more with other docs in audit and https://docs.particular.net/servicecontrol/creating-config-file

```suggestion

This setting is only applicable on instance that use the RavenDB 3.5 storage engine.

The path for the Esent logs on disk.
```
